README.MD

```diff
@@ -162,10 +166,14 @@ cloning and Java file processing.
 
 ### Running AsheAutomation
 
-1. **Execute AsheAutomation:**
+1. **Choose an LLM:**
+    - To use ChatGPT-4, pass `gpt-4` through the arguments.
```

Comment:
Should this be the default? That would make these instructions simpler.

Reply:
We can keep the previous amount of args per entrypoint, but we can also accept an optional added arg (the specified LLM). This would ensure a default of gpt-4, and once new LLMs are added to ASHE, allow users to specify.